MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,138 @@
+## Introduction
+
+Dear maintainer. Thank you for investing the time and energy to help
+make CRIU as useful as possible. Maintaining a project is difficult,
+sometimes unrewarding work. Sure, you will contribute cool features
+to the project, but most of your time will be spent reviewing patches,
+cleaning things up, documenting, answering questions, justifying design
+decisions - while everyone else will just have fun! But remember -- the
+quality of the maintainers work is what distinguishes the good projects
+from the great. So please be proud of your work, even the unglamorous
+parts, and encourage a culture of appreciation and respect for *every*
+aspect of improving the project -- not just the hot new features. 
+
+Being a maintainer is a time consuming commitment and should not be
+taken lightly. This document is a manual for maintainers old and new.
+It explains what is expected of maintainers, how they should work, and 
+what tools are available to them.
+
+This is a living document - if you see something out of date or missing,
+speak up!
+
+## What are a maintainer's responsibility?
+
+Part of a healthy project is to have active maintainers to support the
+community in contributions and perform tasks to keep the project running.
+It is every maintainer's responsibility to:
+
+* Keep the community a friendly place
+* Deliver prompt feedback and decisions on pull requests and mailing
+  list threads
+* Encourage other members to help each other, especially in cases the 
+  maintainer is overloaded or feels the lack of needed expertise
+* Make sure the changes made respects the philosophy, design and
+  roadmap of the project
+
+## How are decisions made?
+
+CRIU is an open-source project with an open design philosophy. This
+means that the repository is the source of truth for EVERY aspect of the
+project. *If it's part of the project, it's in the repo. It's in the 
+repo, it's part of the project.*
```

Comment:
Should we also move the wiki here? ;-)

Reply:
Wiki <s>often</s> sometimes contradicts the code :)